release-notes/aws/v11.3.3.md

```diff
@@ -0,0 +1,11 @@
+## :zap: Giant Swarm Release v11.3.3 for AWS :zap:
+
+**If you are upgrading from 11.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release fixes an issue where `cert-manager` could be killed for exceeding its memory limit. This in turn means that SSL certificates
+wouldn't be automatically renewed and could therefore expire.
+
```

Comment:
```suggestion
Note for SEs: This release contains an external-dns fix introduced in [11.3.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.3.2.md#zap-giant-swarm-release-v1132-for-aws-zap). It requires manual intervention for cluster upgrades in China to work. When upgrading a cluster, existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred. After upgrading, delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets.
```

@sslavic @glitchcrab is this required here?

Reply:
hmm that's a good point - it might be a good idea to retain that